.github/workflows/breaking-changes.yaml

```diff
@@ -0,0 +1,14 @@
+name: (Optional) Detect Breaking Changes
+on: [ pull_request ]
+jobs:
+  go-apidiff:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Install Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: '1.20'
```

Comment:
can you set this to use the one from the go.mod?

Reply:
Our CI/CD is using 1.20: https://github.com/mongodb/atlas-sdk-go/blob/main/.github/workflows/pr.yml#L18

There is a big gap in golang AST between those versions so Golangcilint and other extensions can work more efficiently. There is no impact on using the latest go for this job with actual lowest version of the 

[Offtopic] We could consider bumping version as atlas-cli and terraform seems to use 1.20:

https://github.com/mongodb/mongodb-atlas-cli/blob/master/go.mod#L3